requirements/requirements.txt

```diff
@@ -60,79 +59,77 @@ jaraco.functools==3.0.0   # via irc, jaraco.text, tempora
 jaraco.logging==3.0.0     # via irc
 jaraco.stream==3.0.0      # via irc
 jaraco.text==3.2.0        # via irc, jaraco.collections
-jinja2==2.10.3
+jinja2==2.11.1            # openshift
 jsonschema==3.2.0
-kombu==4.6.7              # via asgi-amqp
-kubernetes==10.0.1        # via openshift
+kubernetes==10.1.0        # via openshift
 lockfile==0.12.2          # via python-daemon
-lxml==4.4.2               # via xmlsec
-markdown==3.1.1
+lxml==4.5.0               # via xmlsec
+markdown==3.2.1
 markupsafe==1.1.1         # via jinja2
-more-itertools==8.1.0     # via irc, jaraco.classes, jaraco.functools, zipp
+more-itertools==8.2.0     # via irc, jaraco.classes, jaraco.functools
 msgpack==0.6.2            # via channels-redis
-msrest==0.6.10            # via azure-keyvault, msrestazure
-msrestazure==0.6.2        # via azure-keyvault
-multidict==4.7.4          # via aiohttp, yarl
+msrest==0.6.11            # via azure-keyvault, msrestazure
+msrestazure==0.6.3        # via azure-keyvault
+multidict==4.7.5          # via aiohttp, yarl
 netaddr==0.7.19           # via pyrad
 oauthlib==3.1.0           # via django-oauth-toolkit, requests-oauthlib, social-auth-core
-openshift==0.10.1
+openshift==0.10.3
 pexpect==4.7.0            # via ansible-runner
 pkgconfig==1.5.1          # via xmlsec
 prometheus-client==0.7.1
-psutil==5.6.7             # via ansible-runner
+psutil==5.7.0             # via ansible-runner
 psycopg2==2.8.4
 ptyprocess==0.6.0         # via pexpect
 pyasn1-modules==0.2.8     # via google-auth, python-ldap, service-identity
 pyasn1==0.4.8             # via pyasn1-modules, python-ldap, rsa, service-identity
-pycparser==2.19           # via cffi
+pycparser==2.20           # via cffi
 pygerduty==0.38.2
-pyhamcrest==1.9.0         # via twisted
+pyhamcrest==2.0.2         # via twisted
 pyjwt==1.7.1              # via adal, social-auth-core, twilio
 pyopenssl==19.1.0         # via twisted
 pyparsing==2.4.6
-pyrad==2.2                # via django-radius
+pyrad==2.3                # via django-radius
 pyrsistent==0.15.7        # via jsonschema
 python-daemon==2.2.4      # via ansible-runner
 python-dateutil==2.8.1    # via adal, kubernetes
 python-ldap==3.2.0        # via django-auth-ldap
 python-memcached==1.59
 python-radius==1.0
-python-string-utils==0.6.0  # via openshift
+python-string-utils==1.0.0  # via openshift
 python3-openid==3.1.0     # via social-auth-core
 python3-saml==1.9.0
 pytz==2019.3              # via django, irc, tempora, twilio
-pyyaml==5.3               # via ansible-runner, djangorestframework-yaml, kubernetes
-redis==3.3.11
+pyyaml==3.13              # via ansible-runner, djangorestframework-yaml, kubernetes
```

Comment:
This is weird - this version number went down?

Reply:
That is strange, I checked `ansible-runner, djangorestframework-yaml, kubernetes` and nothing obvious. They _do_ have a `3` minimum. But you would think the max would be gotten.

Reply:
openshift 0.10.3 (newest) -> kubernetes (10.0.x .. not the newest; the newest version of kubernetes is OK with a newer pyyaml) -> pyyaml about 3.x